client/test/message_store.spec.js

```diff
@@ -110,7 +110,7 @@ describe('Message Store', () => {
       });
     });
 
-    it('isUnread', () => {
+    it.skip('isUnread', () => {
```

Comment:
@aenario : Why have you commented this? Could you had a comment to explain context please? (not specified into commit description)